### ISSUE TITLE
Settings panel

### ISSUE THREAD

**Issue by 3x1t-5tyl3** [3x1t-5tyl3/The-Exitian-MultiTool#27]

**Comment by RobertBaruch** [comment]
Would this also include saving the settings as cloud vars?

**Comment by 3x1t-5tyl3** [comment]
Yes. The idea is to load as much into cloudvars as possible and dynamically loading them on equip.

I currently don't have an issue open mainly because it's *kind of* already implemented. Just not quite finished with it yet.
Together with #23 #24 #25 #26 as those probably will come at once since I'm reworking large parts of the tool at once anyhow.
